packages/components/src/custom-select-control/test/index.js

```diff
@@ -262,6 +271,81 @@ describe.each( [
 		expect( screen.getByRole( 'option', { name: /hint/i } ) ).toBeVisible();
 	} );
 
+	it( 'Should return object onChange', async () => {
+		const user = userEvent.setup();
+		const mockOnChange = jest.fn();
+
+		render(
+			<Component
+				{ ...props }
+				value={ props.options[ 0 ] }
+				onChange={ mockOnChange }
+			/>
+		);
+
+		await user.click(
```

Comment:
Just to confirm, using `@testing-library/user-event` for v1 vs using `@ariakit/test` for legacy v2 is an expected difference, right?